src/UI/PatronUI/PatronsManagementMenu.cs

```diff
@@ -71,26 +71,28 @@ public void AddPatron()
             }
         }
 
-        public void UpdatePatron()
+        public async void UpdatePatron()
         {
-            var existingPatron = UIUtils.DisplaySelectableListResult(

```

Comment:
Why did you delete the implementation of selecting the models by list instead of an attribute? It is supposed that there should not be magic data, so users should not see the IDs of patrons or books